src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/ResultMessage.cs

```diff
@@ -45,6 +50,19 @@ public ResultMessage(string message, bool isError, int? batchId)
             Message = message;
         }
 
+        /// <summary>
+        /// Constructor with flag to indicate query statement completed
+        /// </summary>
+        public ResultMessage(string message, bool isError, int? batchId, bool? hasRowCount)
```

Comment:
You should just merge this in with the constructor above - the `hasRowCount`/`messageType` type param should always be included